server/db/models/models.js

```diff
@@ -1,5 +1,5 @@
 const Sequelize = require('sequelize');
-const { db } = require('./db/index.js');
+const { db } = require('../connection.js');
```

Comment:
requiring db from the same file you are exporting your models and db and declaring your associations... what could go wrong.